src/constants/cctp.ts

```diff
@@ -0,0 +1,8 @@
+import cctpTokens from '../../public/configs/cctp.json';
+
+export const CCTP_TOKEN_NAMES_LOWER_CASE = cctpTokens.map((token) => token.name.toLowerCase());
```

Comment:
CCTP tokens are actually identified via the address not the name in the json.

The name actually refers to the chain name.